README.md

```diff
@@ -1,57 +1,189 @@
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-accordion.svg
+[build-status-url]:https://github.com/moxystudio/react-accordion/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-accordion/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-accordion
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-accordion/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-accordion
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-accordion.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-accordion?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-accordion.svg
 
-{package-drescription}
+A react component that renders an accessible accordion.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-accordion
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+An accordion allows users to toggle the display of sections of content. This package was created to make it easier to implement an accordion with accessible navigation.
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import { Accordion, AccordionItem } from '@moxy/react-accordion';
+
+const MyComponent = () => {
+    <div>
+        <Accordion defaultActiveItems={ [0, 1] } multiple scrollIntoView>
+            <AccordionItem title="Foo">
+                Bar
+            </AccordionItem>
+            <AccordionItem title="Foo1">
+                Bar1
+            </AccordionItem>
+        </Accordion>
+    </div>
+}
+```
 
 ## API
 
-{package-api-description}
+These are the props available in [`@moxy/react-accordion`](https://github.com/moxystudio/react-accordion).
+
+### Accordion Props
+
+#### children
+
+Type: `node` | Required: `true`
+
+One or more `AccordionItem` elements that will be rendered inside the `Accordion`.
+
+#### defaultActiveItems
+
+Type: `array` | Required: `false` | Default: `[]`
+
+The indexes of the items that should be open by default.  
+It is necessary to have the prop `multiple` enabled to have more than one item open.
+
+#### multiple
+
+Type: `boolean` | Required: `false` | Default: `false`
+
+Whether or not multiple items can be open at the same time.
+
+#### className
+
+Type: `string` | Required: `false`
+
+A className to apply to the accordion container.
+
+#### closedClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the accordion container when all the items are closed.
+
+#### onItemToggling
+
+Type: `function` | Required: `false` | Arguments: `index: Number, options: { active: Boolean }`
+
+Callback to be invoked before an item is opened or closed.
+
+### AccordionItem Props
+
+#### title
+
+Type: `any` | Required: `true`
```

Comment:
Is there a specific reason why it's type `any`? Could it be anything but a string?

Reply:
The reason behind this is to give more flexibility, because sometime you might need to pass a component or a function, for example. 

I have an example in the demo where I pass a functional component to the `title`.